ui/pages/settings/alerts-tab/alerts-tab.scss

```diff
@@ -20,6 +20,10 @@
   &__description {
     display: flex;
     align-items: center;
+
+    &__icon {
+      color: var(--color-icon-alternative);
+    }
   }
 
   &__item {
```

Comment:
Bringing the contrast of these icons down to match other info icons

<img width="904" alt="Screen Shot 2022-05-19 at 4 04 55 PM" src="https://user-images.githubusercontent.com/8112138/169418403-d42e422e-ca48-44ec-8f55-53caf969c3ff.png">